proxy/server/target.go

```diff
@@ -38,6 +38,17 @@ import (
 	pb "github.com/Snowflake-Labs/sansshell/proxy"
 )
 
+var (
+	// ReqChanSize is the amount of requests we'll buffer on a given stream
+	// while blocking to do the initial connect. After this the whole stream
+	// will block until it can proceed (or error). Exported as a var so it can
+	// be bound to a flag if wanted. By default this is only one as most RPCs
+	// are unary from the client end so a small buffer is fine. Larger numbers
+	// can cause large explosions in memory usage as potentially needing to buffer
+	// N requests per sub stream that is slow/timing out.
+	ReqChanSize = 1
```

Comment:
maybe ReqBufferSize? the fact that it's used to construct a channel is incidental.

Reply:
Makes sense. Done